articles/whats-wrong-with-i-dont-write-any-tests-i-am-not-a-tester.md

```diff
@@ -0,0 +1,99 @@
+Henri (made up name): "I don’t write any tests since I am not a tester".
+
+I know he handled the situation quite well, but he asked me for advice.
+
+One thing to consider is the underlying question to this, which might be:
+"How do we get people to change their behavior?"
+So here are my thoughts.
+
+## **Yourself First**   
+First of all, think about things to consider for yourself:
+* Why is the practice or tool that you are suggesting any better than the current way of doing things?
+* Can you explain the value in the proposed change?
+* Can you lead by example?
+* Do you have enough patience and skills to teach others?
+I would try to work on yourself before trying to change others.
+    
+ 
+## **Roles?**  
+I see that a lot of people are focused on their role, forgetting the bigger picture of the team and purpose of the work they are doing.
+I would ask these questions:
+* Are we one team that focuses on the Sprint Goal?
+  In a Scrum context there is no “tester”, “programmer” or “architect”, we are all professional engineers that deliver value through collaboration.
+* No matter what, do we stand together and support each other?
+* Are we doing whatever is necessary to deliver a usable product every Sprint?
+ 
+
+## **Done?**  
+Henri: "I don’t write any tests since I am not a tester".  
+Ask: "How do you know when you are done?"  
+  
+What is on our Definition of Done?
+How can we build a usable, tested and fully integrated product increment every Sprint?
+Are we doing that already? Why not?
+   
+   
+## **Fast Automated Feedback**
+Tests send you a message. They send you a message now and in 5 years from now. They tell you:   
+**Is the code working as it is expected or not?**  
+This fast feedback is very valuable if you are working on your product, either by changing it, fixing a bug or by adding a new feature. 
+Ask: "How do you know our product works in 5 years?"    
+  
+
+
+## **Tests are as important as documentation**  
+Documentation is needed, and one good way to document how software **must** work are tests. I emphasize **must**, since written documentation only documents how the software **might** work.  
+We learned too often that documentation gets out of sync too easily.  
+ 
+
+
+
+
+## **Quality**  
+Quality is everyone’s responsibility in a Scrum Team. There is no QA Team in a Scrum context, which means the whole Scrum team is responsible for delivering high quality software that works and is fully tested.  
+  
+Quality attributes that are important:
+* Does it work at all?  
+  Huh!
+* Does it work well?
+* Is it deployed and use-able?  
+  Are the users able to access it?
+* Is it useful? 
+* Is it successful?
+* Does it make the impact we wanted to achieve?  
+   -> Yes! Value is key
+
+   
+## **Tests are code**  
+Are you a coder?? Yes?   
+Tests are code. So write some tests, especially so that you can sleep better at night.  
```

Comment:
Maybe you could spend some words on "test coverage". Some beginners might be frightened by the additional workload generated by writing a lot of tests. More tests are not necessarily better. The focus should be on writing a reasonable amount of tests for code parts that "deserve" being tested automatically. From a regression point of view it's all about managing risk.